.github/workflows/ci.yml

```
@@ -202,7 +202,7 @@ jobs:
     name: 🔗 Integration Tests
     runs-on: ubuntu-latest
     needs: [quality-check, backend-tests, frontend-tests]
-    if: always() && (needs.backend-tests.result == 'success' || needs.backend-tests.result == 'skipped') && (needs.frontend-tests.result == 'success' || needs.frontend-tests.result == 'skipped')
+    if: always() && ((needs.backend-tests.result || 'failure') == 'success' || (needs.backend-tests.result || 'failure') == 'skipped') && ((needs.frontend-tests.result || 'failure') == 'success' || (needs.frontend-tests.result || 'failure') == 'skipped')
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)